### PR TITLE
subresource-loading: Some editorial fixes

### LINKED ISSUE #735: Registered protocol handlers
> If a URL is available from an attached bundle, the browser must retrieve it from the bundle, instead of using any registered custom protocol handler for its scheme.

How does that work given that registered protocol handlers are only for navigations?

### PR DESCRIPTION
**Pull request by hayatoito**
Fix #735, #736, and other editorial issues.

Related issue: #732, which will be fixed in another PR.

### CODE REVIEW

File: explainers/subresource-loading.md [hunk 1]
```diff
@@ -180,13 +179,20 @@ the bundle's URL.
...
+{
+  "source": "./resources.wbn",
+  "resources": ["./iframe1.html", "./iframe2.html"]
+}
+</script>
 
-Note that neither `<script type=importmaps>` nor `<script type=speculationrules>`
-fires a `load` event for an inline rule.
+<iframe src="iframe1.html">
+<iframe src="iframe2.html">
```
Comment (irori) [inline explainers/subresource-loading.md:194]: Well, currently Chromium implementation doesn't allow this (frame source must be `uuid-in-package:`).

File: explainers/subresource-loading.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ a format that allows multiple resources to be bundled, e.g.
 - [Request's destination](#requests-destination)
 - [CORS and CORP for subresource requests](#cors-and-corp-for-subresource-requests)
 - [Content Security Policy (CSP)](#content-security-policy-csp)
+- [Defining the scope](#defining-the-scope)
```
Comment (irori) [inline explainers/subresource-loading.md:21]:
```suggestion
- [Defining the scopes](#defining-the-scopes)
```

Comment (hayatoito) [reply explainers/subresource-loading.md]: Done.

File: explainers/subresource-loading.md (hunk 1 above)
Comment (hayatoito) [reply explainers/subresource-loading.md]: Okay. It would be better not to mention this here. I've reverted this addition. Thanks!

**Comment by hayatoito** [comment]
Thanks for the review. I've addressed the comments.

**Review by irori — COMMENTED** [review]
Should we remove line 166-170 (the sentence mentioning custom protocol handler) per #735?

**Comment by hayatoito** [comment]
> Should we remove line 166-170 of `explainers/subresource-loading.md` (the sentence mentioning custom protocol handler) per #735?

Yes. Done. Thanks!

**Review by irori — APPROVED** [review]
LGTM

**Pull request merged**